app/Console/Commands/InstallCommand.php

```diff
@@ -298,6 +303,33 @@ protected function configureCachet()
     }
 
     /**
+     * Configure the first user.
+     *
+     * @return void
+     */
+    protected function configureUser()
+    {
+        if (!$this->confirm('Do you want to create an admin user?')) {
+            return;
+        }
+
+        // We need to refresh the config to get access to the newly connected database.
+        $this->getFreshConfiguration();
+
+        // Now we need to install the application.
+        $this->call('app:install');
```

Comment:
My laravel core package is left over from the days when I was trying to create an extensible plugin system for a CMS. Harder than I thought it would be at the time. :trollface: The package still remains a kinda odd design. :P